javascript.md

```diff
@@ -77,8 +97,13 @@ Object.keys(person).forEach(lang.hitch(this, function(prop){
 
 - Initial Draft
 
+- Add more general tips.
```

Comment:
Should you add a new date for this? Currently it looks like you made these changes on the 8th.